skimage/morphology/tests/test_convex_hull.py

```diff
@@ -132,10 +132,13 @@ def test_object():
          [1, 0, 0, 0, 0, 0, 0, 0, 0],
          [0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=bool)
 
-    assert_array_equal(convex_hull_object(image, 8), expected8)
+    assert_array_equal(convex_hull_object(image, connectivity=2), expected8)
```

Comment:
The expected variable names should be changed

Reply:
Just saying, Mark - https://github.com/scikit-image/scikit-image/pull/4018/files#diff-4e7cef471759b28ff6d2853a879e6c51R208 🤣.